src/StoreKeeper/WooCommerce/B2C/Exports/OrderExport.php

```diff
@@ -507,6 +587,13 @@ private function getOrderItems($order)
                 $data['attribute_option_ids'] = $attribute_ids;
             }
 
+            $extra = [
+                'wp_row_id' => $orderProduct->get_id(),
+                'wp_row_md5' => md5(json_encode($data, JSON_THROW_ON_ERROR)),
```

Comment:
I meant the md5 of the `$orderProduct` because this is the original row

Reply:
Used `$orderProduct->get_data()`.